docs/conftest.py

```diff
@@ -103,6 +103,11 @@ def show(orig, self, *args, **kwargs):
     return orig(self, *args, *kwargs)
 
 
+@hook(plt)
```

Comment:
where is this function used?

Reply:
`lang/articles/faqs/faq.md`, there are `plt.show()` in docs